CMakeLists.txt

```diff
@@ -744,6 +744,8 @@ elseif(BOOTSTRAPPING_MODE MATCHES "BOOTSTRAPPING.*")
   else()
     set(BOOTSTRAPPING_MODE "HOSTTOOLS")
   endif()
+elseif(BOOTSTRAPPING_MODE)
+  set(SWIFT_EXEC_FOR_SWIFT_MODULES "${CMAKE_Swift_COMPILER}")
```

Comment:
Why don’t we set `SWIFT_NATIVE_SWIFT_TOOLS_PATH` in `build-tooling-libs` instead of adding this `elseif` statement?

Reply:
`build-tooling-libs` does set `SWIFT_NATIVE_SWIFT_TOOLS_PATH` that's why this is needed.

Reply:
Ah, sorry, I misread the `if` above and thought it was negated. I don’t understand why we need this for `build-tooling-libs` but not for normal compilation with `BOOTSTRAPPING_MODE=HOSTTOOLS` though. How does CMake get `SWIFT_EXEC_FOR_SWIFT_MODULES` if I’m in `hosttools` bootstrapping mode?